sim/paladin/protection/protection.go

```diff
@@ -30,7 +30,6 @@ func NewProtectionPaladin(character *core.Character, options *proto.Player) *Pro
 
 	prot := &ProtectionPaladin{
 		Paladin:                         pal,
-		primarySeal:                     protOptions.PrimarySeal,
```

Comment:
Is removing this going to cause the prot paladin to use the wrong primary seal?

Reply:
There is already a `primarySeal` on the Paladin type itself that gets set during Initialize, so the prot/ret specific ones were never used. It's the same options field used in both all cases and the spec-specific ones were just redundant :)

Reply:
Oh gotcha!